guides/doc-Managing_Hosts/topics/proc_managing_and_monitoring_hosts_using_red_hat_web_console.adoc

```diff
@@ -1,23 +1,34 @@
 [[managing_and_monitoring_hosts_using_red_hat_web_console]]
-= Managing and Monitoring Hosts Using Red Hat Web Console
+= Managing and Monitoring Hosts Using {cockpit}
 
-You can access the Red Hat web console web UI through the {Project} web UI and use the functionality to manage and monitor hosts in {Project}.
+You can access the {cockpit} web UI through the {Project} web UI and use the functionality to manage and monitor hosts in {Project}.
 
 .Prerequisites
 
-* Red Hat web console is enabled in {Project}.
-* Red Hat web console is installed on the host that you want to view:
+* {cockpit} is enabled in {Project}.
+* {cockpit} is installed on the host that you want to view:
+
+ifeval::["{build}" != "satellite"]
+** For more information see https://cockpit-project.org/running.html[Running Cockpit].
+endif::[]
+
+ifeval::["{build}" == "satellite"]
 ** For Red{nbsp}Hat Enterprise Linux 8, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/getting-started-with-the-rhel-8-web-console_system-management-using-the-rhel-8-web-console#installing-the-web-console_getting-started-with-the-rhel-8-web-console[Installing the web console] in the _Managing systems using the RHEL 8 web console_ guide.
 ** For Red{nbsp}Hat Enterprise Linux 7, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/managing_systems_using_the_rhel_7_web_console/getting-started-with-the-rhel-web-console_system-management-using-the-rhel-7-web-console#installing-the-web-console_getting-started-with-the-web-console[Installing the web console] in the _Managing systems using the RHEL 7 web console_ guide.
+endif::[]
+
 * {Project} or {SmartProxy} can authenticate to the host with SSH keys. For more information, xref:sect-Managing_Hosts-Establishing_a_Secure_Connection_for_Remote_Commands[].
 
 .Procedure
 
-. In the {Project} web UI, navigate to *Hosts* > *All Hosts* and select the host that you want to manage and monitor with Red Hat web console.
+. In the {Project} web UI, navigate to *Hosts* > *All Hosts* and select the host that you want to manage and monitor with {cockpit}.
 . In the upper right of the host window, click *Web Console*.
 
-You can now access the full range of features available for host monitoring and management, for example, Red Hat Image Builder, through the Red Hat web console.
+You can now access the full range of features available for host monitoring and management, for example, {lorax-composer}, through the {cockpit}.
```

Comment:
Also `lorax-composer` could be `LoraxCompose` expanding to `Lorax Composer` or just `Lorax` because that is the name: https://github.com/weldr/lorax and it could be confused with the command which runs the server component of this project: https://github.com/weldr/lorax/blob/master/docs/lorax-composer.rst